ale/drivers/dawn_drivers.py

```diff
@@ -157,3 +158,21 @@ def focal2pixel_lines(self):
         # Microns to mm
         pixel_size = spice.gdpool('INS{}_PIXEL_SIZE'.format(self.ikid), 0, 1)[0] * 0.001
         return [0.0, 0.0, 1/pixel_size]
+
+    def detector_start_line(self):
+        return 1
+
+    def detector_start_sample(self):
+        return 1
```

Comment:
Just remembered, I just set these to one as placeholder values since they had to be implemented and I was unsure of where to get this information. Where can I get the information for these two methods?

Reply:
It doesn't seem to appear that anything like these exist in the Dawn labels, so I am not entirely sure this needs to be implemented (other than the abstract base class forcing it). @jessemapel Any thoughts?

Reply:
Unless it is the ```VirIrStartXPosition``` and ```VirIrStartYPosition``` (on the VIR sensor, anyway...)

Reply:
Or possibly the ```FirstLine``` and ```FirstLineSample``` of a DawnFC cube...?

Reply:
Wait I just realized when looking in the kaguya driver which I'm currently working on that it has a start_detector_sample which I would assume is the same thing. It gets the information from FIRST_PIXEL_NUMBER in the pds3Label so that might be it

Reply:
Doesn't seem like the dawn labels have this so nevermind I guess?

Reply:
I'd say grab it. The method needs to be implemented either way, right? May as well have the method return something useful.